src/main/io/serial_cli.c

```diff
@@ -2681,7 +2701,23 @@ static void cliMap(char *cmdline)
     cliPrintf("%s\r\n", out);
 }
 
-void *getValuePointer(const clivalue_t *value)
+#ifdef USE_PARAMETER_GROUPS
+static void* getValuePointer(const clivalue_t *var)
+{
+    const pgRegistry_t* rec = pgFind(var->pgn);
+
+    switch (var->type & VALUE_SECTION_MASK) {
+    case MASTER_VALUE:
+        return rec->address + var->offset;
+    case CONTROL_RATE_VALUE:
+        return rec->address + (sizeof(controlRateConfig_t) * getCurrentControlRateProfile()) + var->offset;
```

Comment:
Maybe `controlRateConfig_t` may be handled identically to profile, at least on PG level (and adding support for generic groups)..